### PR TITLE
Update addtohomescreen.css

### PR DESCRIPTION
**Pull request by FlatText**
Removed base64 encodes and added icons to IMAGES folder in root.

Source: http://en.wikipedia.org/wiki/Base64#MIME

"the actual length of MIME-compliant Base64-encoded binary data is usually about 137% of the original data length, though for very short messages the overhead can be much higher due to the overhead of the headers. Very roughly, the final size of Base64-encoded binary data is equal to 1.37 times the original data size + 814 bytes (for headers).

In other words, the size of the decoded data can be approximated with this formula:

bytes = (string_length(encoded_string) - 814) / 1.37"

### CODE REVIEW

**Comment by causefx** [comment]
I dont see a bullets, upload or action image
